interaction.go

```diff
@@ -77,12 +84,13 @@ type MessageInteraction struct {
 }
 
 type CreateInteractionResponseData struct {
-	Content         string              `json:"content"`
-	Tts             bool                `json:"tts,omitempty"`
-	Embeds          []*Embed            `json:"embeds,omitempty"`
-	Components      []*MessageComponent `json:"components"`
-	Attachments     []*Attachment       `json:"attachments"`
-	AllowedMentions *AllowedMentions    `json:"allowed_mentions,omitempty"`
+	Content         string                 `json:"content"`
+	Tts             bool                   `json:"tts,omitempty"`
+	Embeds          []*Embed               `json:"embeds,omitempty"`
+	Components      []*MessageComponent    `json:"components"`
+	Attachments     []*Attachment          `json:"attachments"`
+	AllowedMentions *AllowedMentions       `json:"allowed_mentions,omitempty"`
+	Flags           InteractionMessageFlag `json:"flags,omitempty"`
```

Comment:
Could we instead use the type `MessageFlag` and discord the new constants.

Reply:
Changed to use MessageFlag